api/types/src/move_types.rs

```diff
@@ -1039,8 +1097,8 @@ impl FromStr for EntryFunctionId {
 }
 
 #[inline]
-fn invalid_entry_function_id(s: &str) -> anyhow::Error {
-    format_err!("invalid entry function id {:?}", s)
+fn invalid_entry_function_id<S: Display + Sized>(s: S) -> anyhow::Error {
+    format_err!("invalid entry function id {}", s)
```

Comment:
Can we format this with humans reading this in mind, like `Invalid entry function ID`